packages/devtools_app/flutter_01.log

```diff
@@ -0,0 +1,110 @@
+Flutter crash report.
```

Comment:
please revert these flutter log files. `.log` files should be ignored in our .gitignore file